content/docs/command-reference/config.md

```diff
@@ -238,6 +238,21 @@ to learn more about the state file (database) that is used for optimization.
   When an entry in the database is used (e.g. during the `dvc status`), DVC
   updates the timestamp on that entry. This way, when the database needs a
   cleanup, DVC can sort entries chronologically, and remove the oldest ones.
+  
+- `state.dir` - specify a custom location for the state file directories 
+  (the `links` and `md5s` directory which by default reside in `./dvc/tmp`). 
+  This may be necessary when using DVC on NFS or other mounted volumes.
+  ⚠️ Make sure no other project shares the same `state.dir` location.
```

Comment:
This and the one for `index` are not necessary, as we generate a unique hash based on the path to the repo that is using this, so there is no real danger in multiple projects using the same dirs.
```suggestion
```